pkgs/tools/text/ocrmypdf/default.nix

```diff
@@ -30,14 +30,14 @@ let
 in
 buildPythonApplication rec {
   pname = "ocrmypdf";
-  version = "11.7.3";
+  version = "12.3.0";
   disabled = ! python3Packages.isPy3k;
 
   src = fetchFromGitHub {
     owner = "jbarlow83";
     repo = "OCRmyPDF";
     rev = "v${version}";
-    sha256 = "0gs2w9kl5wwrs0hx2sivq3pdvpf3lkaifblwfbz5g31yl770blji";
+    sha256 = "sha256-gKk49aSgD+mT1Ll+s5bZ2iWUO7onoLwvyX4/Xa8XJNw=";
```

Comment:
```suggestion
    sha256 = "122yv3p0v4fbx30zgppcznwnm7svg97gv0sa103xb6zcld68ggn2";
```

Reply:
Would you mind explaining why this is better than the other format? :sweat_smile: I've just copy pasted the hash given by nix when using `lib.fakeSha256`, why isn't that in the right format?

Reply:
There was a hash mismatch. Format does not matter.

Reply:
Oh. Does github sometimes change the tarballs it provides? Because what I had definitely worked for me originally